packages/angular-cli/commands/serve.ts

```diff
@@ -125,11 +129,6 @@ const ServeCommand = Command.extend({
     return getPort({ port: commandOptions.port, host: commandOptions.host })
       .then((foundPort: number) => {
 
-        if (commandOptions.port !== foundPort && commandOptions.port !== 0) {
```

Comment:
Seems like we _want_ this to throw, since in this case the user asked for a specific port and we should fail if it's unavailable.